### PR TITLE
fix: fix to prevent external filesystem dependency of simulations

### PR DESCRIPTION
**Pull request by da1suk8**
## Description

Fixed to prevent external filesystem dependency of simulation.
The code has been modified as [follows](https://github.com/CosmWasm/wasmd/blob/f95e7c02b3387f00b563ae1ae0c7e5b1e1a5354d/x/wasm/simulation/operations.go#L76-L91).

WeightedOperations returns a function, so I added a test to compare types.

## Motivation and context

Because if the path is not resolved correctly or the file is not found, the call may fail and execution may panic.

## Checklist:

- [x] I followed the [contributing guidelines](https://github.com/line/lbm-sdk/blob/main/CONTRIBUTING.md) and [code of conduct](https://github.com/line/lbm-sdk/blob/main/CODE_OF_CONDUCT.md).
- [x] I have added a relevant changelog to `CHANGELOG.md`
- [x] I have added tests to cover my changes.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated API documentation `client/docs/swagger-ui/swagger.yaml`

### CODE REVIEW

**Review by zemyblue — COMMENTED** [review]
Please add changelog about this PR to `CHANGELOG.md` file

And could you add unittest about this changes.

**Review by zemyblue — COMMENTED** [review]
Please change the title more detail.
I couldn't understand and guess what is change.

File: x/wasm/simulation/operations.go [hunk 1]
```diff
@@ -53,14 +54,19 @@ func WeightedOperations(
...
-	if err != nil {
-		panic(err)
+	var wasmBz []byte
+	if wasmContractPath == "" {
+		wasmBz = testdata.ReflectContractWasm()
+	} else {
+		var err error
+		wasmBz, err = os.ReadFile(wasmContractPath)
+		if err != nil {
+			panic(err)
```
Comment (zemyblue) [inline x/wasm/simulation/operations.go:68]: How about adding the unittest of this lines?

Comment (da1suk8) [reply x/wasm/simulation/operations.go]: This is dead code, so I deleted it.

**Review by zemyblue — COMMENTED** [review]
Please check conflict

**Review by zemyblue — APPROVED** [review]

**Review by tnasu — APPROVED** [review]

**Pull request merged**